index.css

```diff
@@ -108,24 +108,31 @@ section p {
 
 section ul.links {
 	padding: 0;
+	margin-left: 0;
+	margin-right: 0;
 	margin-top: 0;
 	display: -ms-inline-flexbox;
 	display: -webkit-inline-flex;
 	display: inline-flex;
+	-moz-flex-direction:row;
 	-webkit-flex-direction: row;
 	-ms-flex-direction: row;
 	flex-direction: row;
 	-webkit-flex-wrap: wrap;
 	-ms-flex-wrap: wrap;
+	-moz-flex-wrap:wrap;
 	flex-wrap: wrap;
 	-webkit-justify-content: center;
 	-ms-flex-pack: center;
+	-moz-justify-content:center;
 	justify-content: center;
 	-webkit-align-content: flex-start;
 	-ms-flex-line-pack: start;
+	-moz-align-content:center;
 	align-content: flex-start;
 	-webkit-align-items: flex-start;
 	-ms-flex-align: start;
+	-moz-flex-align:start;
 	align-items: flex-start;
```

Comment:
Why? I don't get any of these changes.

PS: We don't support any of the browsers that still need prefixing as their usage is nearly zero.
PPS: Flex box is buggy in IE11 which we do support. Maybe it would be better to convert this to a block of centered inline blocks or something like that.